addons/website_event_exhibitor/models/event_event.py

```diff
@@ -25,6 +26,13 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    def _compute_exhibitor_count(self):
+        data = self.env['event.sponsor'].read_group([('exhibitor_type', '=', 'exhibitor')],
+                                                    ['event_id'], ['event_id'])
+        result = dict((data['event_id'][0], data['event_id_count']) for data in data)
+        for event in self:
+            event.exhibitor_count = result.get(event.id, 0)
```

Comment:
While we are at it it seems ``_compute_sponsor_count`` is missing its domain on self.ids. Could you update it in another commit that we would backport to stable ?

Reply:
I have added a new commit fixing this, however, do we need to create a new branch for this commit to backport it to stable?

Furthermore, wouldn't be a good idea to add a trigger on sponsor_ids in this method as it does not have one?

Reply:
Indeed trigger is missing. Concerning commit, well it can stay here while testing. We will need to backport it using another branch indeed, you may create a branch in stable for that fix (not sure which version it appears, probably at least 14.0).

Reply:
Trigger on 'sponsor_ids' added!

Okay, so we create another branch for this commit when the Task reaches RD Review.